jme3-core/src/main/resources/Common/MatDefs/Misc/Unshaded.j3md

```diff
@@ -59,6 +59,8 @@ MaterialDef Unshaded {
         Float ShadowMapSize
 
         Boolean BackfaceShadows: true
+        
+        Float DesaturationValue

```

Comment:
Should this DesaturationValue be in [0, 1]? 
If so it might be nice to document this on the code.

Reply:
I could add a short comment saying "desaturation value should be in 0-1 range, inclusive" or just "1.0 indicates 100% desaturation" 

I thought of adding a min/max check, although I think it can also work to input a negative desaturation value if you want to increase the saturation, so I left it unbound.

Reply:
"1.0 indicates 100% desaturation" sounds good to me.